testing/test_pytest_html.py

```diff
@@ -12,6 +13,21 @@
 
 pytest_plugins = ("pytester",)
 
+# Force a utf-8 encoding on file io. See
```

Comment:
since this is only for windows can we conditionally execute this if we detect we are on windows ( with [sys.platform](https://docs.python.org/3/library/sys.html#sys.platform) for example ). I'd prefer to avoid modifying builtins unless we have to

Reply:
Yup!